modules/virt-creating-vm-from-template.adoc

```diff
@@ -6,30 +6,34 @@
 [id="virt-creating-vm-from-template_{context}"]
 = Creating a VM from a template
 
-You can create a virtual machine (VM) from a template with an available boot source by using the {product-title} web console.
+You can create a virtual machine (VM) from a template with an available boot source by using the {product-title} web console. You can customize template or VM parameters, such as data sources, Cloud-init, or SSH keys, before you start the VM.
 
-Optional: You can customize template or VM parameters, such as data sources, cloud-init, or SSH keys, before you start the VM.
+You can choose between two views in the web console to create the VM:
+
+* A virtualization-focused view, which provides a concise list of virtualization-related options at the top of the view
+* A general view, which provides access to the various web console options, including Virtualization
 
 .Procedure
 
-. Navigate to *Virtualization* -> *Catalog* in the web console.
-. Click *Boot source available* to filter templates with boot sources.
+. From the {product-title} web console, choose your view:
+** For a virtualization-focused view, select *Administrator* -> *Virtualization* -> *Catalog*.
 +
-The catalog displays the default templates. Click *All Items* to view all available templates for your filters.
-
+** For a general view, navigate to *Virtualization* -> *Catalog*.
+. Click the *Template catalog* tab.
+. Click the *Boot source available* checkbox to filter templates with boot sources. The catalog displays the default templates.
+. Click *All templates* to view the available templates for your filters.
+. To focus on particular templates, enter the keyword in the `Filter by keyword` field.
+. Choose a template project from the *All projects* dropdown menu, or view all projects.
 . Click a template tile to view its details.
 . Optional: If you are using a Windows template, you can mount a Windows driver disk by selecting the *Mount Windows drivers disk* checkbox.
-. If you do not need to customize the template or VM parameters, click *Quick create VirtualMachine* to create a VM from the template.
-+
-If you need to customize the template or VM parameters, do the following:
-
-.. Click *Customize VirtualMachine*.
-.. Expand *Storage* or *Optional parameters* to edit data source settings.
-.. Click *Customize VirtualMachine parameters*.
+. Choose one:
+** If you do not need to customize the template or VM parameters, click *Quick create VirtualMachine* to create a VM from the template.
 +
-The *Customize and create VirtualMachine* pane displays the *Overview*, *YAML*, *Scheduling*, *Environment*, *Network interfaces*, *Disks*, *Scripts*, and *Metadata* tabs.
+** If you need to customize the template or VM parameters, do the following:
```

Comment:
Similar here. The next three steps take place on the tile. Maybe substeps?

. Click a template tile to view its details.
** Optional: If you are using a Windows template, you can mount a Windows driver disk by selecting the *Mount Windows drivers disk* checkbox.
** Optional: If you need to customize the template or VM parameters, do the following:
** Optional: If you need to customize the template or VM parameters, do the following: